paper/paper.md

```diff
@@ -30,57 +30,255 @@ affiliations:
 
 
 # Key concepts
 
+- Conceptual differences between predictions and marginal means?
+- Concept of focal and non-focal terms?
+- For marginalization, we can refer to non-focal terms, meaning it would be good to have the concept introduced somewhere
+
 ## Predictions
 
 At a fundamental level, `modelbased` and similar packages leverage model *predictions*.
+These predictions can be of different types, depending on the model and the question at hand.
+For instance, for linear regressions, predictions can be associated with **confidence intervals** (`predict="expectation"`) or **prediction intervals** (`predict="prediction"`).
+The former corresponds to the uncertainty around the "relationship" (i.e., the conditional estimate, typically of the expectation ($E[X]$) according to a model's parameters)
+while the latter provides information about the range individual observations might take (e.g., _around_ the expectation $E[X]$).
+For generalized linear models (GLMs), <!-- while the distinction between prediction and confidence intervals do not apply, -->
+predictions can be made on the **response scale** (`predict="response"`) or the **link scale** (`predict="link"`).
+This corresponds for instance to predictions in terms of probability (response scale) or log odds (link scale) for logistic regression models.
+
+These different types of estimates can be obtained for observation in the original dataset,
+which is useful to assess the model's goodness-of-fit,
+or for new data (typically a "data grid"), which is useful for visualization and counter-factual reasoning.
+
+For convenience, the `modelbased` package includes 4 related functions, that mostly differ in their default arguments for `data` and `predict`:
+
+- `estimate_prediction()`: original data, prediction intervals.
+- `estimate_expectation()`: original data, confidence intervals.
+- `estimate_relation()`: data grid, predictions on the response scale.
+- `estimate_link()`: data grid, predictions on the link scale.
+
+*Note:* if the defaults are changed, then these functions can become redundant.
+For instance, `estimate_relation(..., predict="link")` will be equivalent to `estimate_link(...)`.
+
+These functions belong to the same family, and their relevance depends on the model and the research question.
+
 
-Mention types of predictions (expectations, predictions, response vs. the link scale)
+## Marginal means and effects
```

Comment:
I think somewhere we can introduce the concept of focal and non-focal terms. For marginalization, we can refer to the wording of "non-focal terms", which is probably better than "non-focal predictors" or "other predictors", because sometimes, we have interaction _terms_.

Reply:
Agreed